.clang-tidy-diff-scans.txt

```diff
@@ -16,6 +16,8 @@
 ./iceoryx_hoofs/test/moduletests/test_concurrent_sofi.cpp
 ./iceoryx_hoofs/test/stresstests/test_stress_sofi.cpp
 ./iceoryx_hoofs/source/relocatable_pointer/relative_pointer_data.cpp
+./iceoryx_hoofs/test/moduletests/test_relative_pointer_data.cpp
+./iceoryx_hoofs/include/iceoryx_hoofs/test/moduletests/test_relocatable_ptr.cpp
```

Comment:
```suggestion
./iceoryx_dust/include/iceoryx_dust/test/moduletests/test_relocatable_ptr.cpp
```

Reply:
@MatthiasKillat Static code analysis [is not necessary to run](https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md#quality-level-1) for quality level 2 classes which are part of `iceoryx_dust`. But it doesn't hurt either :)